src/code/z_shrink_window.c

```diff
@@ -1,25 +1,121 @@
+/**
+ * File: z_shrink_window.c
+ * Description: Draws black top/bottom/side borders on the viewing window (e.g. Z-targeting)
+ */
```

Comment:
```suggestion
/**
 * File: z_shrink_window.c
 * Description: Draws black top/bottom/side borders on the viewing window (e.g. Z-targeting, talking with NPC)
 */
```
While a second example might not be needed, it'd be cool to give a second example with a different sized letter box (I've got some camera-specific shrink window macros on the way)